File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        ArrayList<Integer> lists = new ArrayList<>();
        int n = nums.length;
        for (int i =0;i<n;i++) {
            if(!lists.contains(nums[i])){
                lists.add(nums[i]);
            }
        }
        
        for (int i = 0; i < lists.size(); i++) {
            nums[i] = lists.get(i);
        }

        return lists.size();

        

        
    }
}

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        unique=[]

        for i in nums:
            if i not in unique:
                unique.append(i)


        for i in range(0,len(unique)):
            nums[i]=unique[i]


        return len(unique)